src/docs/Overview.adoc

```diff
@@ -4,9 +4,9 @@
 
 image::architecture.png[align=center]
 
-On the top left is the Studio interface that allows you to build Workflows.
-It can be interactively configured to address specific domains, for instance Finance, Big Data, IoT, Artificial Intelligence (AI). See for instance the Documentation of Machine Learning Open Studio https://www.activeeon.com/public_content/documentation/latest/MLOS/MLOSUserGuide.html[here^], and try it online https://try.activeeon.com/studio/#workflows/templates/machine-learning[here^]. In the middle is the Scheduler that enables an enterprise to orchestrate and automate Multi-users, Multi-application Jobs.
-Finally, at the bottom right is the Resource manager interface that manage and automate resource provisioning
+On the top left is the *Studio* interface that allows you to build Workflows.
+It can be interactively configured to address specific domains, for instance Finance, Big Data, IoT, Artificial Intelligence (AI). See for instance the Documentation of *Machine Learning Open Studio* https://www.activeeon.com/public_content/documentation/latest/MLOS/MLOSUserGuide.html[here^], and try it online https://try.activeeon.com/studio/#workflows/templates/machine-learning[here^]. In the middle is the *Scheduler* that enables an enterprise to orchestrate and automate Multi-users, Multi-application Jobs.
+Finally, at the bottom right is the *Resource manager* interface that manage and automate resource provisioning
```

Comment:
that manages and automates

Reply:
👍 I missed that one @nebilbenmabrouk, well seen!